internal/util/util_test.go

```diff
@@ -157,3 +159,53 @@ func (t *UtilTest) TestResolveConfigFilePaths() {
 	AssertEq(nil, err)
 	ExpectEq(filepath.Join(homeDir, "test.txt"), mountConfig.LogConfig.FilePath)
 }
+
+func (t *UtilTest) TestStringifyShouldReturnAllFieldsPassedInCustomObjectAsMarshalledString() {
+	sampleMap := map[string]int{
+		"1": 1,
+		"2": 2,
+		"3": 3,
+	}
+
```

Comment:
nit: Please remove new line, in unit test we use newline to separate arrange, act and assert statment.

Reply:
done